docs/CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ For more information about `AlexsLemonade/scpca-nf` versions, please see [the re
 <!-- PUT THE NEW CHANGELOG ENTRY RIGHT BELOW THIS -->
 <!-------------------------------------------------->
 
+## UPDATE WITH DATE FOR RELEASE 
+
+* Consensus cell type annotations are now available in all processed `SingleCellExperiment` and `AnnData` objects and merged objects. 
+  * Annotations from `SingleR` and `CellAssign` are used to assign an ontology-aware consensus cell type label. 
```

Comment:
I tried to rephrase this to make a bit more explicitly informational, what do you think?

```suggestion
  * These consensus annotations are ontology-aware consensus cell type labels derived from `SingleR` and `CellAssign` labels. 
```

Reply:
```suggestion
  * The labels obtained from `SingleR` and `CellAssign` are used to assign an ontology-aware consensus cell type label. 
```
I tried a slightly different rephrasing that I think I like a little better.